0-ad-history-mod-package/simulation/templates/structures/han/stable.xml

```diff
@@ -10,6 +10,10 @@
   <Identity>
     <Civ>han</Civ>
     <SpecificName>Mǎ Jiù</SpecificName>
+    <History>
+        Horses were very prevalent in ancient Chinese culture. They symbolized perseverance, resilience, and energy and had an important mythical meaning. Horse statues are abundantly found in graves, with the most famous being the “Flying Horse of Gansu”. This particular statue depicts a Ferghana horse, a breed to which supernatural strength and beauty were attributed. 
+        Horses were fundamental to ancient Chinese society and saw use in many different areas. Stirrups, horse collars, and breast straps are all believed to have been invented in China. Even one of the Han dynasty's nine ministers, the almost highest layer of the government, was solely responsible for the empire's horses: the Minister Coachman. Horses generally were prestige items, though, and very expensive for the average citizen. Besides displaying their wealth, influential Han used them for transportation, either by riding them or having them pull carriages. And because of their high speed, horses were also used for messenging across longer distances. Equestrial sports like hunting or chariot racing were popular in ancient China as well. The game of polo, for example, dates back to the Han dynasty. And most importantly, of course, horses served as mounts for the military.
```

Comment:
What level in the government were the Han ministers? Almost highest, I don't think, works. Were they the third or fourth level?

```suggestion
        Horses were fundamental to ancient Chinese society and saw use in many different areas. Stirrups, horse collars, and breast straps are all believed to have been invented in China. Even one of the Han dynasty's nine ministers, the almost highest layer of the government, was responsible for the empire's horses: the Minister Coachman. Horses generally were prestige items, though, and very expensive for the average citizen. Besides displaying their wealth, influential Han used them for transportation, either by riding them or having them pull carriages. And because of their high speed, horses were also used for carrying messages across long distances. Equestrial sports like hunting or chariot racing were popular in ancient China as well. The game of polo, for example, dates back to the Han dynasty. And most importantly, of course, horses served as mounts for the military.
```

Reply:
Third